client/state/posts/test/selectors.js

```diff
@@ -364,4 +360,30 @@ describe( 'selectors', () => {
 			expect( isRequesting ).to.be.false;
 		} );
 	} );
+
+	describe( '#getPostsBySiteIdAndPostId()', () => {
+		it( 'should return global post id by siteId and postId', () => {
```

Comment:
If we want to add cache tests, I would suggest adding cases for a clean cache, and also other ones to test that the cache break works.